src/content-cli-list.ts

```diff
@@ -1,18 +1,26 @@
-import { PackageCommand } from "./commands/package.command";
-import { SpaceCommand } from "./commands/space.command";
+import {PackageCommand} from "./commands/package.command";
+import {SpaceCommand} from "./commands/space.command";
 
 import commander = require("commander");
+import {contextService} from "./services/context.service";
+import {FatalError, logger} from "./util/logger";
+
 type CommanderStatic = commander.CommanderStatic;
 
-class List {
+export class List {
     public static packages(program: CommanderStatic): CommanderStatic {
         program
             .command("packages")
             .description("Command to list all packages")
             .option("-p, --profile <profile>", "Profile which you want to use to list packages")
             .option("--json", "Return response as json type", "")
+            .option("--includeDependencies", "Include variables and dependencies", "")
             .action(async cmd => {
-                await new PackageCommand().listPackages(cmd.profile, cmd.json);
+                try {
+                    await new PackageCommand().listPackages(cmd.json, cmd.includeDependencies)
+                } catch (err) {
+                    logger.error(new FatalError(err));
```

Comment:
I think we shouldn't throw plain errors to the users, but catch these where thrown (probably api calls) and display more some better information. We can also keep this AI for afterwards.